week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -18,23 +18,44 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+// calling this function should pour your bottle full (volume = 100);
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    this.volume += 100;
   },
+  // calling this function should increase your bottle volume by 10 unit;
   pour: function () {
-    // calling this function should increase your bottle volume by 10 unit;
+    if (this.volume <= 90) {
+      this.volume += 10;
+    }
   },
-  drink: function() {
-    // calling this function should decrease your bottle volume by 10 unit;
+
+  // calling this function should decrease your bottle volume by 10 unit;
+
+  drink: function () {
+    if (this.volume >= 10) {
+      this.volume = this.volume - 10;
+    }

```

Comment:
same for this part of the function. Read carefully the instruction and do it step by step what it's written.

Reply:
Same as above. I use the if condition to make sure it does not go under 0. Any hints so I can fix it?